native_client/setup.py

```diff
@@ -39,4 +39,4 @@ class BuildExtFirst(build):
       license = 'MPL-2.0',
       url = 'https://github.com/mozilla/DeepSpeech',
       ext_modules = [model, utils],
-      install_requires = [ 'numpy', 'scipy==0.19.1' ])
+      install_requires = [ 'numpy' ])
```

Comment:
I expect that to fail during `tc-python-tests.sh` because we won't have `scipy` when trying to run `native_client/client.py`, and it should be the caller's responsability to install scipy.